posthog/api/async_migration.py

```diff
@@ -50,7 +49,7 @@ class Meta:
 
 class AsyncMigrationsViewset(StructuredViewSetMixin, viewsets.ModelViewSet):
     queryset = AsyncMigration.objects.all()
-    permission_classes = [StaffUser]
+    permission_classes = [permissions.IsAuthenticated, permissions.IsAdminUser]
```

Comment:
Unrelated cleanup, there's a DRF global staff user permission.

Reply:
This is borking some tests^ (and some others are failing, related to project & enterprise permissions 👀 )

Reply:
fixed